src/pages/MerchantVoucherDashboard/MerchantVoucherDashboard.tsx

```diff
@@ -46,11 +47,7 @@ const MerchantVoucherDashboard = () => {
   }, []);
 
   const renderAmount = (props: FTRenderProps) => {
-    return Intl.NumberFormat('en-US', {
-      style: 'currency',
-      currency: 'USD',
-      maximumSignificantDigits: 4,
-    }).format(props.value / 100);
+    return currencyFormatter(Number(props.value), 0)
   };
```

Comment:
@jacoblb64 I believe the: 
```javascript
minimumFractionDigits: 0,
maximumFractionDigits: 0
```
below should accomplish the same intent of rounding to the nearest whole dollar, so the logic here should be preserved.